src/Form/EntityEmbedDialog.php

```diff
@@ -571,4 +625,15 @@ public function getDisplayPluginOptions(EmbedButtonInterface $embed_button, Enti
     natsort($plugins);
     return $plugins;
   }
+
+  /**
+   * Registers JS callback that gets entities from entity browser and updates
+   * form values accordingly.
+   */
+  public function registerJSCallback(RegisterJSCallbacks $event) {
```

Comment:
This is not needed any more. Can be removed. See https://github.com/drupal-media/entity_browser/commit/a2fddbbe467049726aca9d4b985f3f3a83c9552d